geopandas/tests/test_plotting.py

```diff
@@ -293,13 +293,13 @@ def test_multipoints(self):
         _check_colors(4, ax.collections[0].get_facecolors(), [MPL_DFT_COLOR] * 4)
 
         ax = self.df2.plot(column="values")
-        cmap = plt.get_cmap()
+        cmap = plt.get_cmap(lut=2)
         expected_colors = [cmap(0)] * self.N + [cmap(1)] * self.N
-        _check_colors(2, ax.collections[0].get_facecolors(), expected_colors)
+        _check_colors(20, ax.collections[0].get_facecolors(), expected_colors)
```

Comment:
Ah, good catch. That's an unfortunate mistake in the tests, since we actually only have 2 geometries in the original `df2` being plotted, but of course what is important here are how many flattened geometries actually got plotted ..